test/test_database.py

```diff
@@ -407,6 +410,26 @@ def test_command_with_regex(self):
         for doc in result["cursor"]["firstBatch"]:
             self.assertTrue(isinstance(doc["r"], Regex))
 
+    def test_cursor_command(self):
+        db = self.client.pymongo_test
+        db.test.drop()
+        db.test.insert_one(SON([("hello", "world"), ("_id", 5)]))
+        db.test.insert_one(SON([("hello2", "world2"), ("_id", 6)]))
+        db.test.insert_one(SON([("hello3", "world3"), ("_id", 7)]))
```

Comment:
This is great but we can do it more efficiently (make tests faster) by using insert_many() to bulk insert multiple documents in one command:
```python
docs = [{"_id": i, "doc": i} for i in range(3)]
db.test.insert_many(docs)
...
actual_docs = list(cursor)
self.assertEqual(docs, actual_docs)
```